src/publish_data/settings/base.py

```diff
@@ -145,7 +145,7 @@
 LOGIN_URL = '/accounts/signin'
 
 
-HOMEPAGE_URL = 'https://data.gov.uk'
+HOMEPAGE_URL = '/'
```

Comment:
Will create an issue to make sure we make this the FQN because we'll need it elsewhere (data.json for instance).